extension/background/background.js

```diff
@@ -9,8 +11,85 @@ chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
             })
         return true
     }
+    if (request.type == 'fetchTimeComments') {
```

Comment:
I believe we should encapsulate it in `fetchChapters`. The content script shouldn't know about the details of how chapters are fetched. So no changes in `content.js` are needed.

Reply:
Okay, I'll make the changes as you suggest.